src/conf_mode/qos.py

```diff
@@ -255,6 +255,9 @@ def verify(qos):
                 if policy_type in ['priority_queue']:
                     if 'default' not in policy_config:
                         raise ConfigError(f'Policy {policy} misses "default" class!')
+                if policy_type in ['rate_control']:
+                    if 'bandwidth' not in policy_config:
+                        raise ConfigError('Bandwidth not defined')
```

Comment:
Should bandwidth be `auto` by default as for other types?
```
# set qos policy shaper foo 
Possible completions:
   bandwidth            Available bandwidth for this policy (default: auto)

# set qos policy random-detect foo 
Possible completions:
   bandwidth            Available bandwidth for this policy (default: auto)

```

Reply:
I use validation instead of default value because it was in 1.3
I think bandwidth is the key parameter for rate-control policy and better when user set is manually

```
vyos@vyos# set qos policy rate-control foo 
Possible completions:
   bandwidth            Available bandwidth for this policy
   burst                Burst size for this class (default: 15k)
   description          Description
   latency              Maximum latency (default: 50)
```